internal/source/compose/v3.go

```diff
@@ -140,11 +157,22 @@ func (c *V3Loader) convertToIR(filedir string, composeObject types.Config, plan
 		if composeServiceConfig.Build.Dockerfile != "" || composeServiceConfig.Build.Context != "" {
 			//TODO: Add support for args and labels
 			// filedir, name, serviceContainer.Image, composeServiceConfig.Build.Dockerfile, composeServiceConfig.Build.Context
-			con, err := new(containerizer.ReuseDockerfileContainerizer).GetContainer(plan, service)
-			if err != nil {
-				log.Warnf("Unable to get containization script even though build parameters are present : %s", err)
-			} else {
-				ir.AddContainer(con)
+
+			// Context path is mandatory, Dockerfile is optional https://docs.docker.com/compose/compose-file/#build
+			composeFileDir := filedir
+			contextPath := filepath.Join(composeFileDir, composeServiceConfig.Build.Context)
+			dockerfilePath := filepath.Join(contextPath, "Dockerfile")
+			if composeServiceConfig.Build.Dockerfile != "" {
+				dockerfilePath = filepath.Join(contextPath, composeServiceConfig.Build.Dockerfile)
+			}
+			if checkForDockerfile(dockerfilePath) {
+				service.ContainerizationTargetOptions = append(service.ContainerizationTargetOptions, dockerfilePath)
```

Comment:
We have to make it available as a containerisation option in plan file itself. Not just in the translate phase.

Reply:
done